scripts/compare_benchmarks.py

```diff
@@ -79,7 +81,7 @@ def calculate_and_format_p_value(old, new):
 diffs = []
 
 table_data = []
-table_data.append(["Benchmark", "prev. iter/s", "runs", "new iter/s", "runs", "change [%]", "p-value (significant if <" + str(p_value_significance_threshold) + ")"])
```

Comment:
What's the reason for not showing the significance threshold anymore?

Reply:
Makes the column shorter. Did you ever look at it?

Reply:
Just asking.